### ISSUE TITLE
Game crashed when open the tech tree

### ISSUE THREAD

**Issue by myclan123** [LoneGazebo/Community-Patch-DLL#10576]
_1. Mod Version (X.Y.Z). Current Version: 4.4.2_

4.2.7

_3. List of Other Mods_

None

_4. Describe the Issue_

Game crashed when open the tech tree

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

[temp.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/13853142/temp.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/13853141/Logs.zip)

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

Open the tech tree

**Comment by azum4roll** [comment]
Is it a consistent crash? Try going to strategic mode before opening it.

**Comment by myclan123** [comment]
Yes, every time it crashed when I open the tech tree. Not 100% happened, but I have encountered it many times. No crash when I didn't open the tech tree at all. I will try the strategic mode next time.

**Comment by azum4roll** [comment]
Tech tree takes up a lot of memory due to all the icons.

**Comment by azum4roll** [comment]
No idea how the game does it, but it's a fact that opening the tech tree is a common cause of running out of memory.

**Comment by axatin** [comment]
you can just try if it crashes also in non-EUI, the tech tree in non-EUI has less icons

**Comment by Nu-Illasera** [comment]
@azum4roll same logic applies as in my previous post, even if you have 1000 ICONS

it will take 60MB RAM / VRAM (see my calculations above).

as for zooming out, its more understandable, due to every tile is modifiable (can have units on it, can have improvements on it, can have a terrain modifier on it (hill, jungle, river, etc)), i assume every tile is unique so needs more VRAM to render (RAM shouldn't make a difference, i assume all tiles in the game are always loaded once the map is generated).

disclaimer once again, i am following common sense, i do not know how the game engine or mods work.
